app/services/recent-events.ts

```diff
@@ -34,6 +33,12 @@ export interface IRecentEvent {
   displayString?: string;
   comment?: string;
   title?: string;
+  createdAt?: string;
```

Comment:
This isn't coming from the server is it?  I see it's used in some hashes, but I didn't see it yesterday when I was working on the timestamps code (not merged yet, blocked by this PR).

Reply:
It's hard for me to say definitively where each property comes from. Each event can originate from io server or streamlabs.com but client side will also fill in missing values. `created_at`, `createdAt` and `CreatedAtTimestamp` properties are present on some but not all events. I only added `createdAt` because it is used in the generation of the cache key used to fetch the read status of `treat` events.

Reply:
In my PR I added another field called something like `iso8601Timestamp`.  I guess the pattern continues.

Reply:
I also used it in the cache key I constructed for merch events, since merch events actually don't have a hash or read property defined.